account_invoice_line_report/report/account_invoice_report_view.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Binh Lam - Komit Consulting <binh.lt@komit-consulting.com>
```

Comment:
You should add the original copyright:

<!-- Copyright 2017 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>